### PR TITLE
Log output of light emission application to file.

### PR DESCRIPTION
**Pull request by tobiaskleiner**
This PR moves the std output from the light emission package/simtel_array into a logfile.

### CODE REVIEW

File: simtools/simtel/simulator_ray_tracing.py [hunk 1]
```diff
@@ -184,7 +184,7 @@ def _make_run_command(
             command += super().get_config_option("mirror_align_random_distance", "0.")
             command += super().get_config_option("mirror_align_random_vertical", "0.,28.,0.,0.")
         command += " " + str(self._corsika_file)
-        command += " 2>&1 > " + str(self._log_file) + " 2>&1"
+        command += f" {self._log_file} 2>&1"
```
Comment (orelgueta) [inline simtools/simtel/simulator_ray_tracing.py:187]: Are you sure about this change? You won't be writing out to the log file this way.

Comment (tobiaskleiner) [reply simtools/simtel/simulator_ray_tracing.py]: Good catch! Just tested this and it is not working here, although the same command works for the light emission package and `logfile 2>&1` is what is mentioned in the simtel_array docs.

**Review by GernotMaier — CHANGES_REQUESTED** [review]
Thanks for this.

Could you add in the corresponding integration tests a check that the log files exists?

```
  INTEGRATION_TESTS:
    - OUTPUT_FILE: <path to check that log files exist>
```

Two minor comments below.

File: simtools/visualization/visualize.py [hunk 2]
```diff
@@ -807,6 +807,9 @@ def plot_simtel_ctapipe(filename, cleaning_args, distance, return_cleaned=False)
     from ctapipe.io import EventSource
     from ctapipe.visualization import CameraDisplay
 
+    logger = logging.getLogger()
+    logger.setLevel(logging.ERROR)
```
[the remark this answers is not in the document]
Comment (tobiaskleiner) [reply simtools/visualization/visualize.py:812]: Removed this and changed how we iterate over events in eventsources without generating a (in this case not needed) log output.

File: simtools/utils/geometry.py [hunk 3]
```diff
@@ -12,6 +12,7 @@
 ]
 
 _logger = logging.getLogger(__name__)
+_logger.setLevel(logging.WARNING)
```
[the remark this answers is not in the document]
Comment (tobiaskleiner) [reply simtools/utils/geometry.py]: Removed.

**Review by GernotMaier — APPROVED** [review]
All good, thank you!

**Pull request merged**